src/clients/shiplift.rs

```diff
@@ -6,7 +6,10 @@ use shiplift::{ContainerOptions, LogsOptions, NetworkCreateOptions, RmContainerO
 use std::fmt;
 
 pub struct Shiplift {
-    client: Docker,
+    // XXX should this be private? Need advice
```

Comment:
Ideally yes, if we manage to not expose any `shiplift` types in our public API then changes to their API will not affect our users :)

Reply:
Which means we could potentially make `patch` / `minor` version releases even if shiplift released a breaking change.

Reply:
We might also consider renaming this client to `Http` or something and completely hide that we are using shiplift under-neath :thinking:

Reply:
Totally agree, making it private. Yeah Shiplift is a temporary name and it does sound awfully specific. I like `Http` or maybe just `Api` or `Engine` (referring https://docs.docker.com/engine/api/). I can clean it up once we agree on the name.

Reply:
Changed to `Http`